### PR TITLE
Update key-vault.md

### PR DESCRIPTION
**Pull request by elraikhm**
Updated for keys and secrets GA release

### CODE REVIEW

File: api/overview/azure/key-vault.md [hunk 1]
```diff
@@ -18,32 +18,44 @@ Read more about [What is Key Vault?](/azure/key-vault/key-vault-whatis) then [Ge
 
 Use the client library to manage keys and related assets such as certificates and secrets.
 
-Install the [NuGet package](https://www.nuget.org/packages/Microsoft.Azure.KeyVault) directly from the Visual Studio [Package Manager console][PackageManager] or with the [.NET Core CLI][DotNetCLI].
+Install the NuGet packages: [Key Vault Keys](https://www.nuget.org/packages/Azure.Security.KeyVault.Keys), 
+[Key Vault Secrets](https://www.nuget.org/packages/Azure.Security.KeyVault.Secrets), [Azure Identity](https://www.nuget.org/packages/Azure.Identity)
```
Comment (kaerm) [inline api/overview/azure/key-vault.md]: I'd also add a line for KV Certificates

File: api/overview/azure/key-vault.md [hunk 2]
```diff
@@ -18,32 +18,44 @@ Read more about [What is Key Vault?](/azure/key-vault/key-vault-whatis) then [Ge
...
+| Key Vault - Secrets | [Azure.Security.KeyVault.Secrets - 4.0.0](https://www.nuget.org/packages/Azure.Security.KeyVault.Secrets/4.0.0) | [ReadMe](https://github.com/Azure/azure-sdk-for-net/blob/Azure.Security.KeyVault.Secrets_4.0.0/sdk/keyvault/Azure.Security.KeyVault.Secrets/README.md) | [Samples](https://github.com/Azure/azure-sdk-for-net/blob/Azure.Security.KeyVault.Secrets_4.0.0/sdk/keyvault/Azure.Security.KeyVault.Secrets/samples) | [Api Reference](https://azuresdkdocs.blob.core.windows.net/$web/dotnet/Azure.Security.KeyVault.Secrets/4.0.0/api/index.html) | [ChangeLog](https://github.com/Azure/azure-sdk-for-net/blob/Azure.Security.KeyVault.Secrets_4.0.0/sdk/keyvault/Azure.Security.KeyVault.Secrets/ChangeLog.md) |
```
Comment (kaerm) [inline api/overview/azure/key-vault.md]: nit: would use README in caps

File: api/overview/azure/key-vault.md [hunk 3]
```diff
@@ -18,32 +18,44 @@ Read more about [What is Key Vault?](/azure/key-vault/key-vault-whatis) then [Ge
...
 
-// sec.Value holds the secret
+// Retrieve a secret using the secret client.
+secret = secret_client.GetSecret(WebConfigurationManager.AppSettings["SecretUri"]);
 ```
+Explore more with the links below
+
+| Service | Package | Readme | Samples | API Reference | Changelog |
+| ------- | ------- | ------ | ------- | ------------- | --------- |
+| Key Vault - Keys | [Azure.Security.KeyVault.Keys - 4.0.0](https://www.nuget.org/packages/Azure.Security.KeyVault.Keys/4.0.0) | [ReadMe](https://github.com/Azure/azure-sdk-for-net/blob/Azure.Security.KeyVault.Keys_4.0.0/sdk/keyvault/Azure.Security.KeyVault.Keys/README.md) | [Samples](https://github.com/Azure/azure-sdk-for-net/blob/Azure.Security.KeyVault.Keys_4.0.0/sdk/keyvault/Azure.Security.KeyVault.Keys/samples) | [Api Reference](https://azuresdkdocs.blob.core.windows.net/$web/dotnet/Azure.Security.KeyVault.Keys/4.0.0/api/index.html) | [ChangeLog](https://github.com/Azure/azure-sdk-for-net/blob/Azure.Security.KeyVault.Keys_4.0.0/sdk/keyvault/Azure.Security.KeyVault.Keys/ChangeLog.md) |
```
Comment (kaerm) [inline api/overview/azure/key-vault.md]: nit: would use README in caps

File: api/overview/azure/key-vault.md [hunk 4]
```diff
@@ -18,32 +18,44 @@ Read more about [What is Key Vault?](/azure/key-vault/key-vault-whatis) then [Ge
...
 ### Example
 
 The following example retrieves the secret for a specific key that is identified in the application settings.
 
 ```csharp
-KeyVaultClient kv = new KeyVaultClient(new KeyVaultClient.AuthenticationCallback(securityToken));
+using System;
+using Azure.Identity;
+using Azure.Security.KeyVault.Secrets;
+using Azure.Security.KeyVault.Keys;
```
Comment (kaerm) [inline api/overview/azure/key-vault.md]: This sample doesn't use Keys I think?

File: api/overview/azure/key-vault.md [hunk 5]
```diff
@@ -18,32 +18,44 @@ Read more about [What is Key Vault?](/azure/key-vault/key-vault-whatis) then [Ge
...
-```bash
-dotnet add package Microsoft.Azure.KeyVault
+Install-Package Azure.Identity
+Install-Package Azure.Security.KeyVault.Keys
+Install-Package Azure.Security.KeyVault.Secrets
 ```
 
 ### Example
 
 The following example retrieves the secret for a specific key that is identified in the application settings.
```
Comment (kaerm) [inline api/overview/azure/key-vault.md]: does this use keys? seems to only be secrets? or no?

File: api/overview/azure/key-vault.md [hunk 6]
```diff
@@ -18,32 +18,44 @@ Read more about [What is Key Vault?](/azure/key-vault/key-vault-whatis) then [Ge
...
+You can install directly from the Visual Studio [Package Manager console][PackageManager] or with the [.NET Core CLI][DotNetCLI].
 
 #### Visual Studio Package Manager
 
 ```powershell
-Install-Package Microsoft.Azure.KeyVault
-```
-
-```bash
-dotnet add package Microsoft.Azure.KeyVault
```
Comment (kaerm) [inline api/overview/azure/key-vault.md]: shouldn't we add these instructions too?

**Comment by scbedd** [comment]
Another update to key-vault.md has been merged. That one also contains the updates you see here (or very close to). Closing this PR.

@elraikhm apologies for the delay of the review and merge on this. I'll be checking the docs repos for outstanding pull requests a couple times a week from now on.

**Pull request closed without merging**